src/fklearn/training/classification.py

```diff
@@ -570,7 +576,7 @@ def lgbm_classification_learner(df: pd.DataFrame,
     features = features if not encode_extra_cols else expand_features_encoded(df, features)
 
     dtrain = lgbm.Dataset(df[features].values, label=df[target], feature_name=list(map(str, features)), weight=weights,
```

Comment:
`categorical_feature='auto'` means treating the columns that have categorical data type in a pandas dataframe as categorical for training. Since this takes the `.values` attribute it will be ignored. I propose removing the `.values` here.